joomla/joomla-cms
# Update Tinymce

#### Summary of Changes

Update tinymce to the latest version
#### Testing Instructions

Check that tnymce works as before and dont forget to test simple, advanced and extended modes which you can choose in the plugin options
